Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+            lower_x += LE_coordinates[0]
+            upper_y += LE_coordinates[1]
+            lower_y += LE_coordinates[1]
+
+            # Construct overall profile coordinates data, 
+            # in-line with required format of MTFLOW            
+            # Check if arrays are sorted and construct the profile coordinates if so
+            upper_x_sorted = np.all((np.diff(upper_x) >= 0)[1:])
+            lower_x_sorted = np.all((np.diff(lower_x) >= 0)[1:])
+
+            if not upper_x_sorted:
+                raise ValueError(f"Upper x-coordinates are not sorted. This indicates an error in the profile generation!")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)